src/pirate/psu.c

```diff
@@ -137,7 +143,11 @@ void psu_measure(uint32_t* vout, uint32_t* isense, uint32_t* vreg, bool* fuse) {
     *isense = ((hw_adc_raw[HW_ADC_CURRENT_SENSE]) * ((500 * 1000) / 4095));
     *vreg = ((hw_adc_voltage[HW_ADC_MUX_VREG_OUT]));
     *vout = ((hw_adc_voltage[HW_ADC_MUX_VREF_VOUT]));
-    *fuse = (hw_adc_raw[HW_ADC_MUX_CURRENT_DETECT] > 300);
+#if (BP_VER == 7)

```

Comment:
As above, at minimum, add comment with `//BUGBUG - TODO - ` so indicate the why ... and so it can then be changed to capability-based.